app/main.py

```diff
@@ -1,6 +1,18 @@
+import os
+import requests
+
+URL = "https://api.weatherapi.com/v1/current.json"
+CITY = "Paris"
+KEY = os.environ.get("API_KEY")
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    data = requests.get(URL + "?" + f"key={KEY}" + "&" + f"q={CITY}").json()
```

Comment:
u can use `params` here, if u want)

Reply:
agree, use params or make this expression a separate constant